docs/hbar_transfer.md

```diff
@@ -0,0 +1,38 @@
+# Transferring Hbar Using the Hedera JSON-RPC Relay
+
+## Add Hedera Network to Metamask
+
+1. Open Metamask and click on the circle in the upper right corner
+2. Select `Settings`->`Networks`->`Add Network`
```

Comment:
Please add a screenshot of what the populated page looks like with the local node as an example

Reply:
done